examples/timeseries.md

```diff
@@ -3,13 +3,15 @@ title: Timeseries
 author: Tom Augspurger
 ---
 
+# Timeseries
+
 Pandas started out in the financial world, so naturally it has strong timeseries support.
 
 The first half of this post will look at pandas' capabilities for manipulating time series data.
 The second half will discuss modelling time series data with statsmodels.
 
 
-```{python}
+```python
```

Comment:
But why? Everything works fine. If you simply need highlighted code without evaluation then you can use the code that would be highlighed in both output and in editor:

`````
@{,eval=False}
```python
print('hello')
```
`````

And this code would be highlighted in the output document only (unless you patch editor highlightings):

`````
```{python, eval=False}
print('hello')
```
`````